src/js/components/Ballot/MeasureItemCompressed.jsx

```diff
@@ -199,33 +203,33 @@ export default class MeasureItemCompressed extends Component {
     // let organizationsToFollowOppose = VoterGuideStore.getVoterGuidesToFollowForBallotItemIdOpposes(measureWeVoteId);
 
     // Voter Support or opposition
-    let is_voter_support = false;
-    let is_voter_oppose = false;
-    let voter_statement_text = false;
+    let isVoterSupport = false;
+    let isVoterOppose = false;
+    let voterStatementText = false;
     const ballotItemSupportStore = SupportStore.get(this.state.ballotItemWeVoteId);
     if (ballotItemSupportStore !== undefined) {
       // console.log("ballotItemSupportStore: ", ballotItemSupportStore);
-      is_voter_support = ballotItemSupportStore.is_support;
-      is_voter_oppose = ballotItemSupportStore.is_oppose;
-      voter_statement_text = ballotItemSupportStore.voter_statement_text;
+      isVoterSupport = ballotItemSupportStore.is_support;
+      isVoterOppose = ballotItemSupportStore.is_oppose;
+      voterStatementText = ballotItemSupportStore.voter_statement_text;
     }
 
     let commentBoxIsVisible = false;
-    if (this.props.showPositionStatementActionBar || is_voter_support || is_voter_oppose || voter_statement_text || this.state.showPositionStatement) {
+    if (this.props.showPositionStatementActionBar || isVoterSupport || isVoterOppose || voterStatementText || this.state.showPositionStatement) {
       commentBoxIsVisible = true;
     }
-    const item_action_bar = (
+    const itemActionBar = (
       <span>
         <ItemActionBar
-          ballot_item_display_name={this.state.ballot_item_display_name}
+          ballotItemDisplayName={this.state.ballotItemDisplayName}
```

Comment:
To make this change you will need to update ItemActionBar since the prop is currently `ballot_item_display_name`.